scripts/ci_test.sh

```diff
@@ -7,6 +7,15 @@ set -o pipefail
 SECRETS_BASE_PATH="${SECRETS_BASE_PATH:-/var/run/secrets}"
 JOB_NAME="assisted-chat-eval-test"
 
+if [[ -n $ASSISTED_CHAT_TEST ]]; then
+    echo "The variable ASSISTED_CHAT_TEST was proided with the value ${ASSISTED_CHAT_TEST}, using it to create the IMAGE and TAG variables for the template"
+else
+    IMAGE_PATH="redhat-user-workloads/assisted-installer-tenant/assisted-chat-test-image-saas-main/assisted-chat-test-image-saas-main"
```

Comment:
Why have a separate variable for IMAGE_PATH? It doesn't look used anywhere else.